tests/functional/api/BackbeatServer.js

```diff
@@ -562,17 +560,11 @@ describe('Backbeat Server', () => {
             getRequest('/_/metrics/crr/all', (err, res) => {
                 assert.ifError(err);
                 const keys = Object.keys(res);
-                assert(keys.includes('backlog'));
```

Comment:
We no longer include backlog in the responses? Wouldn't this be breaking compatibility with Orbit?

Reply:
sorry, I shouldn't have removed this. Both backlog and pending are in the response and data matches